### DISCUSSION TITLE
How to edit metadata (only) in topaz book

### DISCUSSION THREAD

**Discussion by Mark-1976** [apprenticeharper/DeDRM_tools#1805] [Q&A]
Hi, newbie question here, sorry.  I have a couple of topaz books that I purchased from Amazon and have on my Kindle.  They do not appear in the right place in an author sort, on the Kindle, and I want to edit the metadata so as to fix that (e.g., the book appears under "J" when the author is John Smith; most of my books are fine).  I assume I need DeDRM for this.  I have gotten this far: in Calibre 4.23, using DeDRM 6.8.1, I can get Calibre to bring in the file as HTMLZ.  But in this I see numerous typographical errors.  What appears on my Kindle does not have those errors.  I can convert that file to AZW3 and edit the metadata, but when I transfer it back to my Kindle it will not open.  In any case, I want a book that appears the same as it always did, no typos, but with metadata (author name) edited.  Is there any way to do that?  Do I need to strip off the DRM in order to do it?  Thanks.

**Comment by ElleKayEm** [comment]
You have removed the DRM.
